hbase-shell/src/main/ruby/shell/formatter.rb

```diff
@@ -192,6 +199,56 @@ def footer(row_count = nil, is_stale = false)
       end
     end
 
+    # Base abstract class for results formatting.
```

Comment:
Doc on how to do an impl?

Reply:
Good point; I'll add that :smile:

Reply:
I decided to remove the adapter class in the patch update I just pushed. I think it will be better for people to explicitly choose to use TableFormatter.